roles/mediainfo/tasks/main.yml

```diff
@@ -1,10 +1,13 @@
 ---
 # tasks file for mediainfo
-- name: medianinfo | Install mediaserver repository
+- name: mediainfo | download mediainfo repo
+  ansible.builtin.get_url:
+    url: "https://mediaarea.net/repo/deb/repo-mediaarea_1.0-21_all.deb"
+    dest: /tmp/repo-mediaarea_1.0-21_all.deb
+
+- name: mediainfo | Install mediainfo repo
   ansible.builtin.apt:
-    deb: "https://mediaarea.net/repo/deb/repo-mediaarea_{{ mediaarea_repo_version }}_all.deb"
-    state: present
-  when: running_on_server
+    deb: /tmp/repo-mediaarea_1.0-21_all.deb
 
 - name: mediainfo | Install mediainfo
```

Comment:
Huh. Is this the same task as the one above? I mean, does it do anything different?

Reply:
no it isn't. The former installs software that configures the repository at `/etc/apt/sources.list.d/mediaarea.list` which then allows us to install the mediaarea software with the latter. Which reminds me I need to put the update cache so the `apt` updates its repos